packages/jest-cli/src/cli/args.ts

```diff
@@ -515,6 +522,13 @@ export const options = {
       'rare.',
     type: 'boolean',
   },
+  runProjects: {
+    description:
+      'Run only the tests of the specified projects.' +
+      'Jest uses the attribute `displayName` in the configuration to identify each project.',
+    string: true,
+    type: 'array',
```

Comment:
can we say `demand: true` or something here and have `yargs` deal with people not passing any?

Reply:
If I understand what is being requested, you want to force the user to pass at least one element in the array if the `--selectProjects` argument is provided?

I am not too familiar with yargs' `demand` but as far as I know it would force the use of `--selectProjects` itself.

Reply:
Haha, it was so long ago I forgot that I did it already 🤦‍♂️
https://github.com/yacinehmito/jest/blob/af1c2212019008dbf2ddc4174e9c78f3fde734d2/packages/jest-cli/src/cli/args.ts#L54

There's even [a test](https://github.com/yacinehmito/jest/blob/a3a0627020ee71118d71f3e83d314f168eed546f/packages/jest-cli/src/__tests__/cli/args.test.ts#L78).

Reply:
Yes, I know you check it manually, I was wondering if we could pass `demand` and be able to delete that code from here. If `demand` forces the option to be provided, and not just that the options receives values _if_ it was provided, then we shouldn't change 🙂